app/namespace.go

```diff
@@ -145,6 +186,33 @@ func (c *NamespaceClient) parseExistingCerts(ctx *cli.Context) (namespace *names
 	return n, existingCerts, nil
 }
 
+func (c *NamespaceClient) toUserNamespacePermissions(userPermissionsInput map[string]string) ([]*auth.UserNamespacePermissions, error) {
+	var res []*auth.UserNamespacePermissions
+	for email, actionGroup := range userPermissionsInput {
+		u, err := c.authClient.GetUser(c.ctx, &authservice.GetUserRequest{
+			UserEmail: email,
+		})
+		if err != nil {
+			return nil, err
+		}
+		if len(u.GetUser().GetId()) == 0 {
+			return nil, fmt.Errorf("user not found for: %s", email)
```

Comment:
[Nit] consider using multierror so the user can get a list of all the issues that are wrong instead of having to do one at a time